src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -228,7 +229,8 @@ def default_api_version(self):
         ResourceType.MGMT_CONTAINERSERVICE: SDKProfile('2021-05-01', {
             'container_services': '2017-07-01',
             'open_shift_managed_clusters': '2019-09-30-preview'
-        })
+        }),
+        ResourceType.MGMT_CONTAINERINSTANCE: '2021-03-01'
```

Comment:
You cannot define a duplicated resource type `ResourceType.MGMT_CONTAINERINSTANCE` which is the same as above.

Reply:
Multi-version SDK support was determined to be the wrong approach, I have reverted this.